app/src/main/java/com/framgia/wsm/screen/listrequest/ItemListRequestViewModel.java

```diff
@@ -150,11 +157,19 @@ public String getToTime() {
             return mRequest.getCompensation().getToTime();
         }
         if (mRequestOff != null) {
-            if(!mRequestOff.getEndDayHaveSalary().getOffPaidTo().equals("")){
-                return mRequestOff.getEndDayHaveSalary().getOffPaidTo();
+            if (mRequestOff.getEndDayHaveSalary() != null) {
+                if (!"".equals(mRequestOff.getEndDayHaveSalary().getOffPaidTo())) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements